tests/unit/#141.test.scala

```diff
@@ -0,0 +1,10 @@
+// SYNTAX TEST "source.scala"
+
+val language = file.extension {
+//                  ^^^^^^^^^ - keyword.declaration.scala
+val language = file extension {
+//                  ^^^^^^^^^ - keyword.declaration.scala
+val language = file.extension(
+//                  ^^^^^^^^^ - keyword.declaration.scala
+val language = file extension (
+//                  ^^^^^^^^^ - keyword.declaration.scala
```

Comment:
Does this assert its highlighted as a keyword? I expected it to be highlighted the same as “file”

Reply:
The `-` checks that it is not a `keyword.declaration.scala`

Reply:
Aha, I'm not so familiar with the testing infrastructure. Great!

Reply:
I discovered that one today. It must be new, I don't remember seeing it in the first version of the testing infrastructure we use.